app/index.html

```diff
@@ -150,30 +150,38 @@
                     </li>
                 </ul>
                 <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <div>
+                            <form class="navbar-form" style="padding-right: 12px">
+                                <a class="btn btn-small btn-default" href="/automation-dashboard" target="/automation-dashboard"><img src="images/automation_dashboard_30.png" class="right-padding" style="height:25px">Automation Dashboard</a>
+                            </form>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div>
+                            <form class="navbar-form" style="padding-right: 12px">
+                                <a class="btn btn-small btn-default active" href="/studio"><img src="images/studio_30.png" class="right-padding" style="height:25px">Workflow Studio</a>
```

Comment:
maybe add the _target_ to be uniform with the rest <a> tags ?

Reply:
It's the current portal, it needs to open in the current tab. So there is no need to add "target" attribute